MediaBrowser.Providers/Plugins/MusicBrainz/MusicBrainzAlbumProvider.cs

```diff
@@ -256,6 +256,71 @@ public async Task<MetadataResult<MusicAlbum>> GetMetadata(AlbumInfo info, Cancel
             }
         }
 
+        if (Plugin.Instance != null && Plugin.Instance.Configuration.GetMissingTrackInfo)
```

Comment:
This entire section needs some refactoring. The nested indentation levels are too deep and it performs N + N + 3 database lookups. `result.Item.Children` always hits the db.